src/bin/upgrade/main.rs

```diff
@@ -258,34 +253,39 @@ fn get_dependencies(
         .collect::<Result<BTreeMap<_, _>>>()?;
 
     let mut upgrades = DesiredUpgrades::default();
-    for dependency in package
-        .dependencies
-        .clone()
+    for (dependency, old_version) in manifest
+        .get_dependencies()
+        .map(|(_, result)| result.map_err(Error::from))
+        .collect::<Result<Vec<_>>>()?
         .into_iter()
-        .filter(is_version_dep)
```

Comment:
Are we filtering out path dependencies?

Reply:
I could be mistaken, but I think what `is_version_dep` did should now be done by: https://github.com/killercup/cargo-edit/blob/b636aed2643821ecc3397633d60baaebc2f1b8a9/src/bin/upgrade/main.rs#L261-L266

Reply:
That just filters on whether there is a version or not.  You can have a path, or even a git dependency, and have a version field.

Reply:
Added: https://github.com/killercup/cargo-edit/blob/93975ddc2ddb9646c194cc3f9180168e44a22ab5/src/bin/upgrade/main.rs#L261